cbmgmtx/ensurebuckethelper.go

```diff
@@ -15,8 +15,9 @@ type EnsureBucketHelper struct {
 	UserAgent  string
 	OnBehalfOf *cbhttpx.OnBehalfOfInfo
 
-	BucketName string
-	BucketUUID string
+	BucketName  string
+	BucketUUID  string
+	WantMissing bool
```

Comment:
nit: this name was initially a little confusing to me, what about `ensuringDeletion`?